### PR TITLE
queues: add retry / delay flags

### LINKED ISSUE #13370: Describe all options for Wrangler queues commands
### Which Cloudflare product does this pertain to?

Workers

### Existing documentation URL(s)

https://developers.cloudflare.com/workers/wrangler/commands/#queues

### What changes are you suggesting?

As a follow up to 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by elithrar**
Fixes #13370

### CODE REVIEW

**Review by pmiguel — APPROVED** [review]

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -1986,7 +1988,16 @@ wrangler queues consumer add <queue-name> <script-name> [OPTIONS]
...
+- `--batch-size` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+  - Maximum number of messages per batch. Must be a positive integer.
+- `--batch-timeout` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+  - Maximum number of seconds to wait to fill a batch with messages. Must be a positive integer.
+- `--message-retries` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+  - Maximum number of retries for each message. Must be a positive integer.
+- `--max-concurrency` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+  - How long a retried message should be delayed for, in seconds. Must be a positive integer.
+- `--retry-delay` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+  - How long a retried message should be delayed for, in seconds. Must be a positive integer.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md:2000]: `--max-concurrency` and `--retry-delay` have the same description. Is this intentional?

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -1942,6 +1942,8 @@ wrangler queues create <name> [OPTIONS]
 
 - `name` {{<type>}}string{{</type>}} {{<prop-meta>}}required{{</prop-meta>}}
   - The name of the queue to create.
+- `--delivery-delay` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```
Comment (pmiguel) [inline content/workers/wrangler/commands.md]: The final name for this is `--delivery-delay-secs`.

File: content/workers/wrangler/commands.md [hunk 3]
```diff
@@ -1986,7 +1988,16 @@ wrangler queues consumer add <queue-name> <script-name> [OPTIONS]
...
-
+- `--batch-size` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+  - Maximum number of messages per batch. Must be a positive integer.
+- `--batch-timeout` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+  - Maximum number of seconds to wait to fill a batch with messages. Must be a positive integer.
+- `--message-retries` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+  - Maximum number of retries for each message. Must be a positive integer.
+- `--max-concurrency` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+  - The maximum number of concurrent consumer invocations that will be scaled up to handle incoming message volume. Must be a positive integer.
+- `--retry-delay` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```
Comment (pmiguel) [inline content/workers/wrangler/commands.md]: The final name for this is `--retry-delay-secs`

**Review by pmiguel — CHANGES_REQUESTED** [review]
Some updates

File: content/workers/wrangler/commands.md (hunk 2 above)
Comment (elithrar) [inline content/workers/wrangler/commands.md]:
```suggestion
- `--delivery-delay-secs` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```

File: content/workers/wrangler/commands.md (hunk 3 above)
Comment (elithrar) [inline content/workers/wrangler/commands.md]:
```suggestion
- `--retry-delay-secs` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```

**Review by deadlypants1973 — APPROVED** [review]

**Review by Maddy-Cloudflare — APPROVED** [review]

**Review by pmiguel — APPROVED** [review]

**Pull request merged**